build/birt-packages/birt-report-rcp/BIRT.product

```diff
@@ -7,13 +7,18 @@
    </configIni>
 
    <launcherArgs>
-      <vmArgs>-Xms512m
+      <programArgs>-perspective
+org.eclipse.birt.report.designer.ui

```

Comment:
I cannot find the perspective with identifier `org.eclipse.birt.report.designer.ui`
Please double check that we have the definition with such an identifier.